pandas/_typing.py

```diff
@@ -85,6 +86,7 @@
 Label = Optional[Hashable]
 IndexLabel = Union[Label, Sequence[Label]]
 Level = Union[Label, int]
+Shape = Union[int, Tuple[int, ...]]
```

Comment:
Why the union of int and tuple. Shape isn't always a tuple?

Reply:
I think in most cases numpy accepts an int as a 1-tuple

Reply:
I think you're right, pushed the change just now

(Since it needs to be indexable, it may have to be a tuple?)